src/databricks/labs/ucx/assessment/queries/clusters.sql

```diff
@@ -0,0 +1,9 @@
+-- viz type=table, name=Clusters, columns=cluster_id,cluster_name,creator,compatible,failures
+-- widget title=Clusters, col=0, row=15, size_x=3, size_y=4
+SELECT
+  cluster_id,
+  cluster_name,
+  creator,
+  CASE WHEN success=1 THEN "Compatible" ELSE "Incompatible" END AS compatible,
+  failures
+FROM $inventory.clusters
```

Comment:
Add NOT STARTSWITH(cluster_name, "job-") and also change the etl code

Reply:
As a `WHERE` clause? And what ETL code?

Reply:
I am confused where `job-` is used... looking at our E2 env:

![image](https://github.com/databrickslabs/ucx/assets/136075433/bde6ec4c-72bf-4081-8f21-bb3fd5545bc5)

Reply:
yes, in the `WHERE` clause